swagger/v3/component_schemas/NPQApplicationAcceptAttributesRequest.yml

```diff
@@ -1,22 +1,7 @@
 description: "The NPQ Application acceptance request"
 type: object
-required:
```

Comment:
Can we keep those please? as those are required for the acceptance on v3? and add the new funded place